tests/test_setters.py

```diff
@@ -197,6 +197,21 @@ def test_set_unallocated_override_for_radiative_surface_flux(self):
             with self.subTest(name):
                 self._set_unallocated_override_for_radiative_surface_flux(name)
 
+    def test_set_surface_precipitation_rate(self):
+        """Special test since this quantity is not in physics_properties.json file"""
+        state = fv3gfs.wrapper.get_state(
+            names=["total_precipitation", "surface_precipitation_rate"]
+        )
+        total_precip_old = state["total_precipitation"]
+        precip_rate_new = state["surface_precipitation_rate"]
+        precip_rate_new.view[:] = 2 * precip_rate_new.view[:]
+        fv3gfs.wrapper.set_state({"surface_precipitation_rate": precip_rate_new})
+        state_new = fv3gfs.wrapper.get_state(["total_precipitation"])
+        total_precip_new = state_new["total_precipitation"]
+        np.testing.assert_allclose(
+            2 * total_precip_old.view[:], total_precip_new.view[:]
+        )
```

Comment:
Please add an assertion that `precip_rate_new` isn't changed by calling `fv3gfs.wrapper.set_state`.

Reply:
Done.